### PR TITLE
Adding back original installation text

### LINKED ISSUE #54: Add back original installation text (the one which includes emojis from Carlos's script) as we have already validated it with users

### PR DESCRIPTION
**Pull request by psschwei**
# Changes

- Restores original install text from the KonK script, and adds a little formatting to some other messages

/kind enhancement

Fixes #54

### CODE REVIEW

File: pkg/install/install.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var eventingVersion = "0.24.1"
 // Kourier installs Kourier networking layer from Github YAML files
 func Kourier() error {
 
-	fmt.Println("Starting Kourier Networking layer " + kourierVersion + " install...")
+	fmt.Println("🕸️ Instlling Kourier networking layer...")
```
Comment (csantanapr) [inline pkg/install/install.go:33]: I really want to see the versions being printed

Comment (csantanapr) [inline pkg/install/install.go:33]:
```suggestion
	fmt.Println("🕸️ Installing Kourier networking layer...")
```

File: pkg/install/install.go [hunk 2]
```diff
@@ -95,14 +95,14 @@ spec:
...
-	fmt.Println("Finished installing Networking layer")
+	fmt.Println("    Finished installing Kourier Networking layer")
 
 	return nil
 }
 
 // Serving installs Knative Serving from Github YAML files
 func Serving() error {
-	fmt.Println("Starting Knative Serving " + servingVersion + " install...")
+	fmt.Println("🍿 Installing Knative Serving...")
```
Comment (csantanapr) [inline pkg/install/install.go:105]: let's keep the version printed. 🙏

**Review by csantanapr — CHANGES_REQUESTED** [review]
I think we should keep the versions printed

**Comment by psschwei** [comment]
> I think we should keep the versions printed

sounds good to me

**Comment by csantanapr** [comment]
perfect
/lgtm

**Review by csantanapr — APPROVED** [review]

**Pull request merged**